src/page/template/modal/group-creation.htm

```diff
@@ -1,15 +1,15 @@
 <div id="group-creation-modal" data-bind="with: $root.groupCreation" class="group-creation__modal">
-  <modal params="isShown: isShown, onClosed: afterHideModal">
+  <modal params="isShown: isShown, onClosed: afterHideModal, ariaLabelby: 'group-creation-label', lock: true">
```

Comment:
Seems like you never consume the `lock: true` param. Am I missing something?

Reply:
Good point, I've missed this one during code cleanup after previous implementations.